onediff_diffusers_extensions/onediffx/lora/lora.py

```
@@ -32,15 +33,31 @@
 class OneDiffXWarning(Warning):
     pass
 
 
 warnings.filterwarnings("always", category=OneDiffXWarning)
+warnings.filterwarnings("always", category=DeprecationWarning)
+
+def deprecated():
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)